test/api/test_dataset_collections.py

```diff
@@ -1,4 +1,5 @@
 import json
+import time
 import tarfile
```

Comment:
Travis complains about the import order here (assuming you added the changes to this file on purpose).